packages/babel-preset-css-prop/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can't leave a comment on the diff but `packages/babel-plugin-jsx-pragmatic/tsconfig.json` being empty looks to be the cause for [the Flow CI issue](https://github.com/emotion-js/emotion/pull/2360/checks?check_run_id=2504489068).